### ISSUE TITLE
Actively manage subcard

### ISSUE THREAD

**Issue by Praxie** [Pliohub/Plio#1307]
**This issue is for Milestone 5a-3!**

![screen shot 2018-08-17 at 2 56 12 pm](https://user-images.githubusercontent.com/2095940/44265056-b9d87980-a22d-11e8-8d76-f2290e0b810f.png)

- After adding the first document of a kind, a top-level subcard should be appended under the Actively manage subcard.
- After the last document of the top-level subcard (e.g. Risks) is deleted or unlinked from the canvas item, the subcard should disappear.
- There's also an idea to change the behaviour of the Delete button in subcards. It should provide a user with 2 options: to unlink the document or remove it completely (though it should go to the Deleted screen if there is such option).
<img width="719" alt="screen shot 2018-08-23 at 1 41 39 am" src="https://user-images.githubusercontent.com/2095940/44494750-b7cf4a00-a675-11e8-8d76-e2ff93146842.png">

**Comment by sives** [comment]
I would reword slightly as follows:  "Do you wish to unlink the nonconformity "Missed calls" from the current standard, or delete it completely?"

Button options are Cancel, Unlink, Delete

**Comment by Serhiy91** [comment]
https://github.com/Pliohub/Plio/pull/1382
https://github.com/Pliohub/Plio/pull/1406
https://github.com/Pliohub/Plio/pull/1414
https://github.com/Pliohub/Plio/pull/1446
https://github.com/Pliohub/Plio/pull/1447
https://github.com/Pliohub/Plio/pull/1455
https://github.com/Pliohub/Plio/pull/1460
https://github.com/Pliohub/Plio/pull/1479
https://github.com/Pliohub/Plio/pull/1491
https://github.com/Pliohub/Plio/pull/1493

**Comment by Praxie** [comment]
@sives sorry but we have to keep the phrase "current document" instead of more specific "current standard". This component is re-usable and should be maintainable. Is it a problem?

**Comment by sives** [comment]
No it's not a problem
